src/main/groovy/edu/oregonstate/mist/videogamesampleapi/resources/PlatformResource.groovy

```diff
@@ -0,0 +1,77 @@
+package edu.oregonstate.mist.videogamesampleapi.resources
+
+import edu.oregonstate.mist.api.AuthenticatedUser
+
+/**
+ * Created by george on 12/22/15.
```

Comment:
Line 6 doesn't add extra information. This information can be found via git log.